autosubmit/profiler/profiler.py

```diff
@@ -1,48 +1,47 @@
-#!/usr/bin/env python3
```

Comment:
Changes in this file are only cosmetic. Adjusted the header to be a single comment block with the license, fixed IDE warnings, and imported just `Process` from `psutil`, instead of the whole module/namespace.